Add `validators-graffiti-client-append-format` reference doc (#566)
Consensys/doc.teku: docs/reference/cli/index.md

This option takes precedence over [`--validators-graffiti`](#validators-graffiti).

### `validators-graffiti-client-append-format`

<Tabs>
  <TabItem value="Syntax" label="Syntax" default>

```bash
--validators-graffiti-client-append-format=<STRING>
```

  </TabItem>
  <TabItem value="Example" label="Example" >

```bash
--validators-graffiti-client-append-format=CLIENT_CODES
```

  </TabItem>
  <TabItem value="Environment variable" label="Environment variable" >

```bash
TEKU_VALIDATORS_GRAFFITI_CLIENT_APPEND_FORMAT=CLIENT_CODES
```

  </TabItem>
  <TabItem value="Configuration file" label="Configuration file" >

```bash
validators-graffiti-client-append-format: CLIENT_CODES
```

  </TabItem>
</Tabs>

> Appends CL and EL clients information with a space to user's graffiti. On

Reviewer: ```suggestion
Appends consensus layer and execution layer clients' information with a space, to the user's graffiti. On
```

> a separate BN/VC setup should be set on a Beacon Node. This feature helps

Reviewer: ```suggestion
Configure the beacon node and validator client setup on a separate Beacon Node. This feature helps
```

Author: No, it means if you run it separately, you should add a flag or change it only on BN. It will not affect VC settings. If you run single node, you do it on this single node, no questions.

developers and community to analyze client diversity and block anomalies.

> Default value is `AUTO`.

Reviewer: ```suggestion
The default value is set to `AUTO`.
```

> Following options are available:

Reviewer: ```suggestion
The following options are available:
```

> - `AUTO`: If user's graffiti is empty sets graffiti to reveal

Reviewer: Maybe this can be rewritten in a slightly different way to  for improve clarity, for example:

`AUTO`: If a user's graffiti is empty, it automatically updates to reveal information about the consensus and execution clients. This includes their codes and build commits, for example `TK508459f2BUbb9ba13c`, with the structure as follows:
   - `TK` represents the Teku consensus client.
   -  `508459f2` is the Teku build commit.
   -  `BU` indicates the Besu execution layer client.
   -  `bb9ba13c` displays the Besu build commit in a 4-byte hexadecimal format.

Author: I like it, updated
